### ISSUE TITLE
enum + nullable: true doesn't compute the good type

### ISSUE THREAD

**Issue by RoXuS** [acacode/swagger-typescript-api#145]
Hey,

In case of enum + `nullable: true`, we get an union types without `null` type.

Example:

```json
{
  "openapi": "3.0.1",
...
  "MyDto": {
        "type": "object",
        "properties": {
          "id":  {
            "type": "integer",
            "format": "int64"
          },
          "legalCategory": {
            "type": "string",
            "nullable": true,
            "enum": [
              "SARL",
              "ASSOCIATION"
             ]
           }
      }
```

```ts
export interface MytDto {
  id: number;
  legalCategory:
    | "SARL"
    | "ASSOCIATION";
...
```

**Comment by js2me** [comment]
Hello @RoXuS !   
Thanks for this helpful bug report!   
Will fix it at next near release

**Comment by js2me** [comment]
Hello @RoXuS !   
This bug should be fixed in 4.3.0

**Comment by RoXuS** [comment]
Thx!
